### PR TITLE
done film page

### PR DESCRIPTION
**Pull request by shapikso**

### CODE REVIEW

File: src/movie.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        <div class="title">
+            <span>Lord of the Rings: The Return of the King</span>
+        </div>
+        <div class="movie-details" style="
+         background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url('https://image.tmdb.org/t/p/original/lXhgCODAbBXL5buk9yEmTpOoOgR.jpg');
+         background-size: cover;">
+            <div class="short-info base-gaps">
+                <div class="basic-link space-between">PG-13</div>
+                <div class="basic-link space-between">201m</div>
+                <div class="basic-link space-between">popularity: 80/100</div>
```
Comment (duhenets) [inline src/movie.html]: these are not links what to use the class "basic-link"

File: src/movie.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            </div>
+            <span class="movie-header">Genres</span>
+            <div class="genres base-gaps">
+                <div class="basic-btn space-between">Fantasy</div>
+                <div class="basic-btn space-between">Advanture</div> 
+                <div class="basic-btn space-between">Action</div>
```
Comment (duhenets) [inline src/movie.html]: these are not buttons what to use the class "basic-btn"

File: src/movie.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    <div class="title">
+        <span>{{title}}</span>
+    </div>
+    <div class="movie-details" style="
+     background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url('{{backdrop_path}}');
+     background-size: cover;">
+        <div class="short-info base-gaps">
+            <div class="basic-link space-between">{{adult}}</div>
+            <div class="basic-link space-between">{{runtime}}</div>
+            <div class="basic-link space-between">popularity: {{popularity}}/100</div>
```
Comment (duhenets) [inline src/movie.html]: "space-between" - this is a bad name for the class

File: src/movie.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+            <div class="overview base-gaps">
+                <p class="overview-text">Aragorn is revealed as the heir to the ancient kings as he,
+                 Gandalf and the other members of the broken fellowship struggle to save Gondor from Sauron's forces. Meanwhile, 
+                 Frodo and Sam take the ring closer to the heart of Mordor, the dark lord's realm.</p>
+            </div>
+        </div>
+    </div>
+</main>
+
+<script type="text/template" id="MovieTemplate">
```
Comment (duhenets) [inline src/movie.html]: camelCase for id name

File: src/styles/movie-page.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    border-radius: 10px;
+}
+
+.space-between {
+    height: 39px;
+    min-width: 20px;
+}
+.base-gaps {
+    display: flex;
+   margin: 50px;
```
Comment (duhenets) [inline src/styles/movie-page.css]: fix indent
why such a large margin from above?

File: src/styles/movie-page.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
+.movie-details {
+    width: 1420px;
```
Comment (duhenets) [inline src/styles/movie-page.css]: maybe a percentage relative to the parent

File: src/styles/movie-page.css [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    height: 650px;
+    border: 1px solid rgba(255, 255, 255, 0.3);
+    box-sizing: border-box;
```
Comment (duhenets) [inline src/styles/movie-page.css:5]: this property is already in reset.css

File: src/movie.html [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+            <div class="genres base-gaps">
+                <div class="basic-btn space-between">Fantasy</div>
+                <div class="basic-btn space-between">Advanture</div> 
+                <div class="basic-btn space-between">Action</div>     
+            </div>
+            <span class="movie-header">Overview</span> 
+            <div class="overview base-gaps">
+                <p class="overview-text">Aragorn is revealed as the heir to the ancient kings as he,
+                 Gandalf and the other members of the broken fellowship struggle to save Gondor from Sauron's forces. Meanwhile, 
+                 Frodo and Sam take the ring closer to the heart of Mordor, the dark lord's realm.</p>
```
Comment (duhenets) [inline src/movie.html:49]: if there is too much text, it falls out of the block

File: src/movie.html [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+        <span class="movie-header">Genres</span>
+        <div class="genres base-gaps">
+
+        </div>
+        <span class="movie-header">Overview</span> 
+        <div class="overview base-gaps">
+            <p class="overview-text">{{overview}}</p>
+        </div>
+</script>
+<script type="text/template" id="GenreTemplate">
```
Comment (duhenets) [inline src/movie.html]: camelCase for id name

**Review by duhenets — CHANGES_REQUESTED** [review]
Check the comments

File: src/movie.html [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+    </div>
+</header>
+
+<main class="main">
+    <div class="container movie-wrapper">
+        <div class="title">
+            <span>Lord of the Rings: The Return of the King</span>
+        </div>
+        <div class="movie-details" style="
+         background: linear-gradient( rgba(0, 0, 0, 0.700), rgba(0, 0, 0, 0.700) ), url('https://image.tmdb.org/t/p/original/lXhgCODAbBXL5buk9yEmTpOoOgR.jpg');
```
Comment (ratroian) [inline src/movie.html]: styles in style.css

File: src/movie.html [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+        </div>
+    </div>
+</main>
+
+<script type="text/template" id="movieTemplate">
+    <div class="title">
+        <span>{{title}}</span>
+    </div>
+    <div class="movie-details" style="
+     background: linear-gradient( rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7) ), url('{{backdrop_path}}');
```
Comment (ratroian) [inline src/movie.html]: styles in style.css

File: src/movie.html [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+                <p class="overview-text">Aragorn is revealed as the heir to the ancient kings as he,
+                 Gandalf and the other members of the broken fellowship struggle to save Gondor from Sauron's forces. Meanwhile, 
+                 Frodo and Sam take the ring closer to the heart of Mordor, the dark lord's realm.</p>
+            </div>
+        </div>
+    </div>
+</main>
+
+<script type="text/template" id="movieTemplate">
+    <div class="title">
```
Comment (ratroian) [inline src/movie.html:56]: if title - maybe use tag <h...>?

File: src/movie.html [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+        <span>{{title}}</span>
+    </div>
+    <div class="movie-details" style="
+     background: linear-gradient( rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7) ), url('{{backdrop_path}}');
+     background-size: cover;">
+         <div class="base-gaps" id="info">
```
Comment (ratroian) [inline src/movie.html]: maybe ul?

File: src/movie.html [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+            <div class="small-info info-block-size">{{adult}}</div>
```
Comment (ratroian) [inline src/movie.html]: li

**Review by ratroian — APPROVED** [review]
Please, check comments

**Review by ratroian — APPROVED** [review]

**Review by duhenets — APPROVED** [review]

**Pull request merged**